packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -175,6 +204,15 @@ export const EditTopicModal = ({
           })}
           isDisabled={!!topic.archived_at}
         />
+        <div className="char-error-row">
+          <CWText type="caption">Character count: {characterCount}/250</CWText>
+          <MessageRow
+            // @ts-expect-error <StrictNullChecks/>
```

Comment:
// @ts-expect-error <StrictNullChecks/> can be removed, pass
`statusMessage={descErrorMsg || ''}`